Library/Homebrew/rubocops/checksum_cop.rb

```diff
@@ -0,0 +1,55 @@
+require_relative "./extend/formula_cop"
+
+module RuboCop
+  module Cop
+    module FormulaAudit
+      class Checksum < FormulaCop
+        def audit_formula(_node, _class_node, _parent_class_node, body_node)
+          %w[Stable Devel HEAD].each do |name|
+            next unless spec_node = find_block(body_node, name.downcase.to_sym)
+            _, _, spec_body = *spec_node
+            audit_checksums(spec_body, name)
+            resource_blocks = find_all_blocks(spec_body, :resource)
```

Comment:
Auditing the stable spec audits everything which is outside those blocks (as they are implicitly stable). The existing code is therefore fine, it's just this code that needs adjusted for that implicit stable resource handling. Make sense?